src/Features/RemotePaymentMethods/DefaultPaymentGateways.php

```diff
@@ -58,6 +58,30 @@ public static function get_all() {
 					self::get_rules_for_cbd( false ),
 				),
 			),
+			array(
+				'key'        => 'mollie',
```

Comment:
I know we're not entirely sure on the path forward here, but maybe we should update this to `mollie_wc_gateway_banktransfer` or one of the other Mollie gateways so the payment gateway REST API doesn't 404.

We could probably even use this gateway to provide the help text needed for this gateway.

Reply:
Resolved here https://github.com/woocommerce/woocommerce-admin/pull/7092/commits/6557c4d2ba76c3bea7d40adf8999bb406c1c52f2